.travis.yml

```diff
@@ -0,0 +1,8 @@
+language: java
+jdk:
+    - oraclejdk8
```

Comment:
For the time being, Travis CI does not offer OpenJDK 8, only OpenJDK 7. If we want to use Java 8, we have to go with OracleJDK 8 for Travis.

I run the bot with OpenJDK 8. But I understand your suggestion, we should run the bot and test it with the same version or in multiple versions in any case.

Reference: https://docs.travis-ci.com/user/languages/java/